src/function/matrix/reshape.js

```diff
@@ -33,7 +33,7 @@ export const createReshape = /* #__PURE__ */ factory(name, dependencies, ({ type
    *
    * @param {Array | Matrix | *} x  Matrix to be reshaped
    * @param {number[]} sizes        One dimensional array with integral sizes for
-   *                                each dimension
+   *                                each dimension. -1 as wildcard allowed.
```

Comment:
Can you explain the wildcard a bit more? (I.e. "You can specify the wildcard -1 to automatically calculate the size" or so).

Can you also add an example in the comments above?

Reply:
I just wanted to wait for your feedback on the functionality itself (if you even want to add it to the library).  
I'll add comments and documentation then.